content/docs/dispensary/prescriptions.md

```diff
@@ -374,10 +341,7 @@ To edit a prescription line, tap on it. You will be presented with the `Edit Ite
 </div>
 
 1. Open the Prescription you want to edit.
-2. Tap on the line you want to edit. An identical window to `Add Item` appears. At this stage:
-
-   - Edit the main `Issue` field
-   - or change the number of packs value at the batch number level
+2. Tap on the line you want to edit.
```

Comment:
I think this is enough? People will get the picture pretty quickly?

Reply:
The entire docs is way overkill IMHO. Yes, they should get the picture. I honestly don't know who this is supposed to be written for.

Reply:
Yeah I agree, feels like we're over invested in click the `Add new` button to add a new item type docs.
Not very helpful IMHO.

I think things like warnings why they show up, why you can and can't do things is much more useful...